core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.spec.ts

```diff
@@ -48,6 +49,9 @@ describe('DotEmaShellComponent', () => {
         component: DotEmaShellComponent,
         imports: [RouterTestingModule, HttpClientTestingModule],
         detectChanges: false,
+        firstChild: new ActivatedRouteStub({
```

Comment:
This is to mock the firstChild
https://github.com/ngneat/spectator/issues/328

Reply:
ohh good to know.